addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -128,6 +128,8 @@ def action_in_progress(self):
         else:
             self.write({'state': 'in_progress'})
         # Set the sequence number regarding the requisition type
+        if self.company_id:
+            self = self.with_company(self.company_id)
```

Comment:
```suggestion
        self = self.with_company(self.company_id)
```
`company_id` will be defined
https://github.com/odoo/odoo/blob/9ea2bdb3e8401b1450936c1635fdcc5cd82df03a/addons/purchase_requisition/models/purchase_requisition.py#L61

Reply:
didn't see there was a default